Makefile

```diff
@@ -27,8 +31,12 @@ help: ##@miscellaneous Show this help.
 	@perl -e '$(HELP_FUN)' $(MAKEFILE_LIST)
 
 all: ##@build Build the final app.zip from scratch
-all: node_modules clean compile-contracts patch-ipfs mk-build-dir copy-misc copy-backend compile-js copy-frontend archive
+all: node_modules clean compile-contracts patch-ipfs mk-build-dir copy-misc copy-backend compile-js copy-frontend archive install-build
```

Comment:
I don't know if I like this here. Granted it is after `archive`, but if somehow `node_modules` end up in the ZIP intended for ElasticBeanstalk deployment it will break the installation on the EB hosts.

Reply:
OK, so how best do we make sure that `node_modules` is there whenever we're building for `localhost` but not for `development` or `production`? That would be truly ideal, and then I could happily bounty lots of cool work :tada:

Reply:
Eh, whatever, I can't think of any clean way of achieving that, so fuck it.

Reply:
Ummm, ok. How about, if `localhost` is set, check for `full-build`. If it exists, overwrite the `back-end` and `frontend` dirs (should leave `node_modules` in tact). If it doesn't exist, `mkdir` and follow the same process as for `development` or `production`? Is that possible?